src/components/clusterConfiguration/ReviewCluster.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { Table, TableBody, TableVariant } from '@patternfly/react-table';
-import Humanize from 'humanize-plus';
 import { Cluster, Host, Inventory, stringToJSON } from '../../api';
 import { getSimpleHardwareInfo } from '../hosts/hardwareInfo';
 import { DetailList, DetailItem } from '../ui/DetailList';
 import { ClusterValidations, HostsValidations } from './ReviewValidations';
 
 import './ReviewCluster.css';
+import { fileSize } from '../hosts/utils';
```

Comment:
please move this above css import

Reply:
fixed in 7e34b60df66f4d71cc2458fa63ff557d02d1bf7f